tests/unit/core/client/test_base.js

```diff
@@ -1,10 +1,24 @@
+import createAmoStore from 'amo/store';
```

Comment:
Do you really need the `amo` store here?

Reply:
Yes, because I want to cover [`store.getState().api.requestId`](https://github.com/mozilla/addons-frontend/pull/6926/files#diff-4ddfaf1de1a8550c255cf55df60dbf62R66) with real actions and reducers. The `client/base` file doesn't have Flow so this needs to stay in sync. I can try to convert it to Flow real fast.

Reply:
> I can try to convert it to Flow real fast.

It won't be fast so I'm not going to bother with it. It touches all of the gnarly dynamic locale code.